src/site/development-assets/scss/styles.scss

```diff
@@ -0,0 +1,195 @@
+/*****************************************************
+ * Compiled from SCSS to CSS using gulp. If you would
```

Comment:
Are these files from me doing something? They are supposed to ge genereated i thought?